cli/src/commands/account.rs

```diff
@@ -389,11 +414,75 @@ mod tests {
         );
 
         let vanity = None;
-        let account = Account::New { network: 0, seed, vanity, discreet: false };
+        let account = Account::New { network: 0, seed, vanity, discreet: false, save_to_file: None };
         let actual = account.parse().unwrap();
         assert_eq!(expected, actual);
     }
 
+    #[test]
+    fn test_new_save_to_file() {
+        let dir = TempDir::new().expect("Failed to create temp folder");
+        let file = dir.path().join("my-private-key-file").to_string_lossy().to_string();
```

Comment:
```suggestion
        let file = dir.path().to_owned().push("my-private-key-file");
```
this would retain strong path typing

Reply:
Fixed in 98b68f456976a582b851cfba02c24e359f38c6f6 :sunny: